tiledb/sm/array_schema/array_schema.h

```diff
@@ -754,3 +756,7 @@ class ArraySchema {
 }  // namespace tiledb
 
 #endif  // TILEDB_ARRAY_SCHEMA_H
+
+/** Converts the filter into a string representation. */
+std::ostream& operator<<(
+    std::ostream& os, const tiledb::sm::ArraySchema& schema);
```

Comment:
This declaration should be inside the `#endif`. In this case it's mostly harmless, but the deviation from pattern hinders readability.